web/static/map/map.js

```diff
@@ -37,6 +37,7 @@ export function initMap() {
   urlVars.level = urlVars.level || Cookies.get('last_level') || '1';
   urlVars.tags = urlVars.tags || Cookies.get('last_tags');
   urlVars.fixable = urlVars.fixable || Cookies.get('last_fixable');
+  urlVars.errorId = urlVars.errorId || Cookies.get('error_id');
```

Comment:
Do the `errorId` URL parameter name seem correct ? As it will be used on RSS feeds and other "persistent" resources I think it must be choosen wisely.

Reply:
Should be `marker_uuid`

Reply:
Fixed in 1cd2f26f94ab6a6c3092f6cc6e84c6145d01da3b.